lighthouse-cli/test/smokehouse/pwa-config.js

```diff
@@ -42,7 +42,8 @@ module.exports = {
   {
     gatherers: [
       'dobetterweb/domstats',
-      'http-redirect'
+      'http-redirect',
+      'start-url',
```

Comment:
definitely need some expectations for the smoke tests as this is going to be a tricky one :)